server/routes/skills.router.js

```diff
@@ -1,16 +1,22 @@
 const express = require('express');
+const { rejectUnauthenticated } = require('../modules/authentication-middleware');
 const pool = require('../modules/pool');
 const router = express.Router();
 
-router.delete('/user/:id/:skill', (req, res) => {
-  console.log('deleting skill', req.params.id, req.params.skill);
-  const queryText = `DELETE FROM "user_skills" WHERE "user_id" = $1 AND "skill_id" = $2;`;
-  pool.query(queryText, [req.params.id, req.params.skill])
-  .then((result) => res.send(result))
-  .catch((error) => {
-    console.log('error in deleting skill', error);
-    res.sendStatus(500);
-  })
+router.delete('/user/:id/:skill', rejectUnauthenticated, (req, res) => {
```

Comment:
Does this delete route work? Only asking because it has /: twice. Not sure how the params work here.